src/main/locales/da/bf_locale.h

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
As these are new files to Betaflight, and have no history with Cleanflight - you can use the ./DEFAULT_LICENCE.md contents.

Reply:
Right, no issue